iphone/Maps/UI/Settings/MWMSettingsViewController.mm

```diff
@@ -196,7 +180,23 @@ - (void)configNavigationSection {
       break;
   }
   [self.nightModeCell configWithTitle:L(@"pref_map_style_title") info:nightMode];
+}
+
+- (void)show3dBuildingsAlert:(UITapGestureRecognizer *)recognizer {
+  UIAlertController *alert =
+  [UIAlertController alertControllerWithTitle:L(@"pref_map_3d_buildings_title")
```

Comment:
Usually indent of 4 spaces is used in these cases. Without it, the line looks like a new, separate operator.

Reply:
In this file, I don't change anything except moving this chunk from one method into another (NightMode from one section to another). 
Should I fix the formatting on the file In this commit?

Reply:
It's ok to fix some minor issues if you're already editing that file. And avoid copy-pasting the entropy )